maplibre_gl/lib/src/controller.dart

```diff
@@ -747,6 +747,108 @@ class MapLibreMapController extends ChangeNotifier {
     return _maplibrePlatform.getTelemetryEnabled();
   }
 
+  /// Sets the maximum frames per second for the map rendering.
+  ///
+  /// This can help optimize performance on lower-end devices by limiting
+  /// the rendering frequency.
+  ///
+  /// The returned [Future] completes after the change has been made on the
+  /// platform side.
+  Future<void> setMaximumFps(int fps) async {
+    return _maplibrePlatform.setMaximumFps(fps);
+  }
+
+  /// Forces the map to use online mode, disabling any offline functionality.
+  ///
+  /// This is useful for testing or when you want to ensure the map always
+  /// uses the latest data from the network.
+  ///
+  /// The returned [Future] completes after the change has been made on the
+  /// platform side.
+  Future<void> forceOnlineMode() async {
+    return _maplibrePlatform.forceOnlineMode();
+  }
+
+  /// Eases the camera to a new position with an optional duration.
+  ///
+  /// The [cameraUpdate] specifies the target camera position, and [duration]
+  /// specifies the animation duration in milliseconds (optional).
+  ///
+  /// The returned [Future] completes with true if the animation finished successfully,
+  /// or false if it was cancelled.
+  Future<bool> easeCamera(CameraUpdate cameraUpdate, Duration? duration) async {
```

Comment:
Use the named parameter for `duration` to ensure consistency with the code of other platform specific methods
```suggestion
  Future<bool> easeCamera(CameraUpdate cameraUpdate, {Duration? duration}) async {
```